### DISCUSSION TITLE
[Question][Hw6] Autograder timeout error

### DISCUSSION THREAD

**Discussion by snurf198** [prosyslab-classroom/cs348-information-security#478] [Q&A]
Name: Sangmin Lee
<img width="939" alt="image" src="https://github.com/prosyslab-classroom/cs348-information-security/assets/55132820/9a3cbf40-978a-4a25-854a-6ece4b44134d">

When I submit my code, after about 50 minutes, I get this kind of error. Since it takes so many times to grade once, it is very hard for me to debug why this kind of error is out. Can I have any idea why I am getting this kind of issue?

**Comment by KAIST-JongchanPark** [comment]
@snurf198 I regrade your submission and it looks working well. Please check your result.

**Comment by JaejunHan** [comment]
@KAIST-JongchanPark
I suffer the same problem. Could you help me?

**Comment by yeonhee-ryou** [comment]
@JaejunHan you can trigger it by yourself

**Comment by JOYB28** [comment]
> @JaejunHan you can trigger it by yourself

Do you mean resubmit button? ('다시 재출')
I have the same problem and tried to resubmit but always get this result.

<img width="1115" alt="스크린샷 2023-05-29 19 25 01" src="https://github.com/prosyslab-classroom/cs348-information-security/assets/29791719/69937be7-3488-4132-82e7-dd2b06ebefd8">

**Comment by m-spitfire** [comment]
@JOYB28 if resubmission doesn't solve it, it might be because at some point of your fuzzing process you error out, i.e. return non-zero code. Maybe try running the fuzzer for a bit and see if it runs fine for 10–15 minutes.
